app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -528,6 +528,15 @@ bool shouldConsiderPoseAsValid(const XrPosef& pose) {
 }
 #endif
 
+static XrHandJointEXT GetHandJointIndexForAim() {
```

Comment:
Why do we need a function for this ? Wouldn't be more appropriate to use  ``` #define``` directives instead ? 

```
#if defined(PICOXR)
  #define XR_JOINT_PALM XR_HAND_JOINT_WRIST_EXT
#else
  #define XR_JOINT_PALM XR_HAND_JOINT_PALM_EXT
#endif
```

Reply:
that's an interesting, I don't really mind